### ISSUE TITLE
Incorrectly requires main to be an array

### ISSUE THREAD

**Issue by tfausak** [hdgarrood/bower-json#1]
[The `bower.json` spec](https://github.com/bower/spec/blob/d912c3cea0515ec2573641fb14fb57ab6ac91b8a/json.md#main) says that the "main" attribute can be either a string or an array of strings. This package requires it to be an array of strings, which causes it to reject valid `bower.json` files. 

I ran into this problem while using `psc-publish`. Here's the error message I got:

```
There is a problem with your package, which meant that it could not be     
published.                                                                 
Details:                                                                   
  The bower.json file could not be parsed as JSON:                         
    aeson reported: BadSchema [ObjectKey "main"] (WrongType TyArray (String
    "src/Batteries.purs"))                                                 

  Please ensure that your bower.json file is valid JSON.                   
```

**Comment by hdgarrood** [comment]
Thanks for reporting. Not sure how I missed this.

**Comment by hdgarrood** [comment]
Fixed by 318d54174eef20d8c621ac1a280ab61c969146f8.
